dockerfiles/base/scripts/base/startup.sh

```diff
@@ -251,20 +255,40 @@ init() {
   source "${SCRIPTS_CONTAINER_SOURCE_DIR}"/cli.sh
 }
 
+cleanup() {
+  RETURN_CODE=$?
+
+  # CLI developers should only return '3' in code after the init() method has completed.
+  # This will check to see if the CLI directory is not mounted and only offer the error
+  # message if it isn't currently mounted.
+  if [ $RETURN_CODE -eq "3" ]; then
```

Comment:
The other thing we could do, @riuvshin - is not even bother with a :/cli volume mount. Instead, we could always have the cli.log written to :/data volume mount which is the root mount that is always required.